micro_pumas_v1.F90

```diff
@@ -1200,28 +1100,51 @@ subroutine micro_pumas_tend ( &
   !$acc      copyout (qcsinksum_rate1ord,tlat,qvlat,qctend,qitend,nctend,     &
```

Comment:
change `!$acc               mg_snow_props) &` to `!$acc               mg_snow_props,proc_rates) &`

Reply:
added